### ISSUE TITLE
Popup displayed after changing visualization to cities while using the visitor map

### ISSUE THREAD

**Issue by quba** [piwik/piwik#8061]
An error message appears after selecting the cities icon on the bottom left of the Visitor Map. Please see:

![screenshot 2015-06-08 09 16 12](https://cloud.githubusercontent.com/assets/647328/8029570/0e7c8ff0-0dbf-11e5-9f18-f2a41ccdd3c3.png)

While using IE9 there's a popup message displayed. While using Chrome I can see a warning in the browser's console.

**Comment by mattab** [comment]
Thanks for the report

**Comment by tsteur** [comment]
I issued a pull request after debugging the issue but I'm not really sure what I'm doing there. It's more like a guess and seems to work but I have no idea whether it causes any side effects.

The alert is triggered within kartograph itself if `console` is not defined and I'd have to modify kartograph itself to prevent it from triggering alerts.

BTW:
We're currently using `0.5.2` of kartograph.js . We should maybe update to the latest version `0.8.7` as some point to make sure it works in all browsers and to get the performance tweaks that were made etc. I tried to update to latest version but it results in errors and don't know how much work it is to fix it.
